eslint-default.js

```diff
@@ -4,42 +4,45 @@ module.exports = {
     project: ["tsconfig.json"],
     ecmaVersion: 2018, // Allows for the parsing of modern ECMAScript features
     sourceType: "module", // Allows for the use of imports
-    ecmaFeatures: {
-      jsx: true, // Allows for the parsing of JSX
-    },
   },
-  plugins: ["@typescript-eslint", "sonarjs"],
+  plugins: ["@typescript-eslint", "sonarjs", "unused-imports"],
   extends: [
     "eslint:recommended",
     "plugin:@typescript-eslint/recommended",
     "plugin:@typescript-eslint/recommended-requiring-type-checking",
-    "plugin:react/recommended",
-    "plugin:sonarjs/recommended",
     "plugin:eslint-comments/recommended",
-    "prettier", // Uses eslint-config-prettier to disable ESLint rules from @typescript-eslint/eslint-plugin that would conflict with prettier
+    "plugin:jest/recommended",
+    "plugin:sonarjs/recommended",
     "plugin:prettier/recommended", // Enables eslint-plugin-prettier and displays prettier errors as ESLint errors. Make sure this is always the last configuration in the extends array.
+    "prettier", // Uses eslint-config-prettier to disable ESLint rules from @typescript-eslint/eslint-plugin that would conflict with prettier
   ],
   rules: {
     curly: ["error", "all"],
+    "no-console": "warn",
     "no-unneeded-ternary": "error",
-    "no-redeclare": "off", // warns on typescript function overrides
-    //"@typescript-eslint/no-redeclare": ["error"], // this should be enabled, but causes 'Definition for rule '@typescript-eslint/no-redeclare' was not found'
 
-    "react/prop-types": "off",
-    "react/display-name": "off",
-    "react/jsx-curly-brace-presence": ["error", "never"],
-    "react/react-in-jsx-scope": "off",
+    "prettier/prettier": [
```

Comment:
Opravdu si tohle přečte i prettier, který formátuje kód? (ne jen eslint plugin pro spolupráci s prettierem) 

Jde mi o to, jestli můžeme bezpečně smazat prettier-config.js

Reply:
Ve VS Code ano. Ale je pravda, že v IntelliJ jsem to nezkoušel. Výhoda je, že si nemusím do projektu tahat krom tohohle configu i prettier. My jsme právě na NTT tento config nepoužívali.